src/Microsoft.DotNet.Interactive/KernelCommandExtensions.cs

```diff
@@ -33,6 +33,13 @@ public static class KernelCommandExtensions
             }
         }
 
+        public static void OverrideId(
```

Comment:
This should not be public. Also, why have this method instead of a setter?

Reply:
removed